### ISSUE TITLE
Add option to run a one-off task based on an existing task-definition

### ISSUE THREAD

**Issue by martinzugnoni** [brunocascio/ecs-deploy#2]
`ecs-deploy` has an option to run one-off tasks using an existing task definition.
https://github.com/fabfuel/ecs-deploy#running-a-task

I'm particularly using those to run `collectstatic` and `migrate` commands in my django project after the ECS deploy is done, like this:
```
$ ecs run <cluster> <task-definition> -c <container> "django-admin collectstatic --noinput"
```

Is there any easy way to add support for that?

**Comment by brunocascio** [comment]
Hey @martinzugnoni!

I have just published (pre-release) [2.0.0 version](https://github.com/brunocascio/ecs-deploy/releases/tag/v2.0.0)

[Here](https://github.com/brunocascio/ecs-deploy/tree/v2.0) the docs about how to use now and the [changes involved](https://github.com/brunocascio/ecs-deploy/pull/3)

Let me know if it works

**Comment by martinzugnoni** [comment]
👍  thank you!
